CHANGELOG.md

```diff
@@ -57,9 +58,162 @@
+into the main Kong Gateway (OSS) repository. We are making a gradual transition, starting with the
+grpc-gateway plugin first:
+
+- Moved grpc-gateway inside the Kong repo [#7466](https://github.com/Kong/kong/pull/7466)
+
+### Additions
+
+#### Core
+
+- Control Planes can now send updates to new Data Planes even if the Control Planes lose connection to the database
+  [#6938](https://github.com/kong/kong/pull/6938)
+- Kong now automatically adds `cluster_cert`(`cluster_mtls=shared`) or `cluster_ca_cert`(`cluster_mtls=pki`) into
+  `lua_ssl_trusted_certificate` when operating under Hybrid mode. Before, Hybrid mode users needed to configure
+  `lua_ssl_trusted_certificate` manually as a requirement for Lua to verify the Control Plane’s certificate
+  [#7044](https://github.com/kong/kong/pull/7044)
+- New `declarative_config_string` option allows loading declarative config directly from a string
+  [#7379](https://github.com/kong/kong/pull/7379)
+
+#### PDK
+
+- Accept tables in response body for stream subsystem
+  [#7082](https://github.com/kong/kong/pull/7082)
+
+#### Plugins
+
+- **hmac-auth**: add support for the "@request-target" field support
+  [#7037](https://github.com/kong/kong/pull/7037)
+- **syslog**: Add facility configuration capability
+  [#6081](https://github.com/kong/kong/pull/6081).
+  Thanks, [jideel](https://github.com/jideel)!
+- **Prometheus**: Expose dataplane status on control plane, new metrics data_plane_last_seen, data_plane_config_hash and data_plane_version_compatible
+  https://github.com/Kong/kong-plugin-prometheus/pull/98
+- **Zipkin**: has now service.name and route.name tags
+  https://github.com/Kong/kong-plugin-zipkin/pull/115
+
+#### Hybrid Mode
+
+- Expose upstream healthchecks endpoint on status API
+  [#7429](https://github.com/Kong/kong/pull/7429)
+- Control Planes are more lenient when checking Data Planes' compatibility
+  [#7488](https://github.com/Kong/kong/pull/7488)
+- Groundwork for Hybrid Mode 2.0 Protocol has been started. This code isn't active by default in Kong 2.5, but it allows future development.
+  [#7462](https://github.com/Kong/kong/pull/7462)
+
+
+### Fixes
+
+#### Core
+
+- `select_by_cache_key` does not do unnecessary cache reads in `off` strategy
+  [#7146](https://github.com/kong/kong/pull/7146)
+- Kong can handle errors that happen inside a plugin's `init_worker` handler
+  [#7099](https://github.com/kong/kong/pull/7099)
+
+
+#### Hybrid Mode
+
+- Control planes don't perform health checks upon CRUD upstreams/targets events
+  [#7085](https://github.com/kong/kong/pull/7085)
+- Fixed a bug that provoked unnecessary cache flips on Data Plane
+  [#7112](https://github.com/kong/kong/pull/7112)
+- TLS keepalive request no longer can share their context
+  [#7102](https://github.com/kong/kong/pull/7102)
+- HTTP Status 405 is now handled by Kong's error handler
+  [#6933](https://github.com/kong/kong/pull/6933).
```

Comment:
Those two does not seems to be Hybrid mode related changes, maybe move them under "Core".